ccc/deliverydb.py

```diff
@@ -1,18 +1,32 @@
-from deliverydb.db import DeliveryDB
+import typing
+
+import psycopg
+
+from deliverydb.db import DeliveryDB, delivery_db_no_orm
 import model.compliancedb
 
 
 def default_with_db_cfg(
     db_cfg: model.compliancedb.ComplianceDbConfig,
     hostname: str = None,
-) -> DeliveryDB:
+    orm: bool = True,
```

Comment:
as a general stmt, it is very rarely a good idea to use boolean flags to switch between different flavours or variants (there are more ORMs besides SQLAlchemy, also there even are IIRC more postgresql-drivers than psycopg).

So - if at all - I would prefer to use an enum. However, I would not actually suppose this to be a good idea either way.

I would rather recommend to factor-out the (supposedly) common functionality between creating a SQLAlchemy session, and a psycopq-connection. Which I assume would be to create a db-connection-URL from a db-cfg object.

Also (yes, this is not part of this PR), hostname should by default be read from db-cfg. I understand you introduced this for local development. at the very least, this ought rather be called sth. like `overwrite_hostname`, and only be used if passed. even better: accept a full db-connection-URL. then everything can be overwritten.